system/Helpers/text_helper.php

```diff
@@ -672,3 +672,22 @@ function excerpt(string $text, ?string $phrase = null, int $radius = 100, string
         return str_replace('  ', ' ', $ellPre . $prev . $phrase . $post . $ellipsis);
     }
 }
+
+if (!function_exists('str_contains')) {
+    /**
+     * Determine if a given string contains in a given substring.
+     *
+     * @param  string  $haystack
+     * @param  string|string[]  $needles
+     * @return bool
+     */
+    function str_contains(string $haystack, string $needle): bool
+    {
+        if ($haystack !== '' && $needle !== '' && strpos($haystack, $needle) !== false) {
```

Comment:
Since this is polyfilling PHP 8's `str_contains()`
```php
var_dump(str_contains('', '')); // true
var_dump(str_contains('foo', '')); // true
```

But this implementation will return `false` for both.